prompt_toolkit/contrib/completers/base.py

```diff
@@ -12,7 +12,7 @@ class WordCompleter(Completer):
     """
     Simple autocompletion on a list of words.
 
-    :param words: List of words.
+    :param words: List of words, or a callable that produces a list of words.
```

Comment:
For consistency with some other parts of prompt_toolkit, I think it's better to have a second parameter named ``get_words``.